reporter/src/funTest/kotlin/reporters/StaticHtmlReporterFunTest.kt

```diff
@@ -38,15 +38,13 @@ import org.ossreviewtoolkit.utils.ORT_NAME
 import org.ossreviewtoolkit.utils.test.patchExpectedResult
 import org.ossreviewtoolkit.utils.test.readOrtResult
 
-private val HOW_TO_FIX_TEXT_PROVIDER: HowToFixTextProvider = object : HowToFixTextProvider {
-    override fun getHowToFixText(issue: OrtIssue): String? {
-        return """
-            * *Step 1*
-            * __Step 2__
-            * ***Step 3***
-            ```Some long issue resolution text to verify that overflow:scroll is working as expected.``` 
-        """.trimIndent()
-    }
+private val HOW_TO_FIX_TEXT_PROVIDER: HowToFixTextProvider = HowToFixTextProvider {
```

Comment:
Maybe drop the explicit type here as `: HowToFixTextProvider = HowToFixTextProvider` reads a bit verbose?

Reply:
Done.